app/src/main/java/com/phlox/tvwebbrowser/activity/main/dialogs/settings/MainSettingsView.kt

```diff
@@ -60,6 +62,14 @@ class MainSettingsView @JvmOverloads constructor(
         }
     }
 
+    private fun initKeepScreenOnUI() {
+        vb.scKeepScreenOn.isChecked = settingsModel.keepScreenOn
+
+        vb.scKeepScreenOn.setOnCheckedChangeListener { buttonView, isChecked ->
```

Comment:
Parameter 'buttonView' is never used, could be renamed to _

Reply:
I was unsure about the coding style here, since `initAdBlockConfigUI` also writes out unused parameters.

At the company I work at we do this as well. Mainly to avoid (_, __, ___, ...) (when several parameters are unused) and so that later no one has to look up again what parameters are passed if the line should be extended later.  We therefore essentially move it to the highlighting of the IDE, which grays out unused parameters to make it *visible*.

Reply:
I also consider renaming unused variables to _ not a good practice. Although sometimes I also do this just to calm down the annoying IDE hints))